examples/standard/http-redirect-rewrite/httproute-rewrite-prefix-path.yaml

```diff
@@ -8,7 +8,11 @@ spec:
   hostnames:
```

Comment:
Can't leave a comment far enough up, but do you mind updating the "used in" section for this file? It looks like the current value is no longer used + the list does not contain the redirect and rewrite guide this is referenced in.

Reply:
@robscott 
Done